app-client/src/main/resources/application-docker.properties

```diff
@@ -0,0 +1,12 @@
+#SERVER CONFIGURATIONS
+server.port= 8081
+server.servlet.context-path= /cdc
```

Comment:
I think having `context-path` is not required in a single deployment

Reply:
Resolved